app/queries/get_content_collection.rb

```diff
@@ -1,11 +1,12 @@
 module Queries
   class GetContentCollection
-    attr_reader :content_format, :fields
+    attr_reader :content_format, :fields, :locale
 
-    def initialize(content_format:, fields:, publishing_app: nil)
+    def initialize(content_format:, fields:, publishing_app: nil, locale: nil)
```

Comment:
Yes @tuzz, that's exactly why I used a "double" default value. 
I didn't make it mandatory in order to keep the former calls of the `initialize` method valid.